offchain-modules/packages/x/src/monitor/bridge-metric.ts

```diff
@@ -13,14 +13,14 @@ export type txTokenInfo = {
 export class BridgeMetricSingleton {
   private static instance: BridgeMetricSingleton;
 
-  private relayBlockHeightNum: Prometheus.Gauge<any>;
-  private relayBridgeTxNum: Prometheus.Counter<any>;
-  private relayBridgeTokenAmountNum: Prometheus.Gauge<any>;
-  private relayForkHeightNum: Prometheus.Gauge<any>;
+  private readonly relayBlockHeightNum: Prometheus.Gauge<string>;
+  private readonly relayBridgeTxNum: Prometheus.Counter<string>;
+  private readonly relayBridgeTokenAmountNum: Prometheus.Gauge<string>;
+  private readonly relayForkHeightNum: Prometheus.Gauge<string>;
 
-  private relayErrorLogNum: Prometheus.Gauge<any>;
+  private readonly relayErrorLogNum: Prometheus.Gauge<string>;
```

Comment:
please confirm that string is valid

Reply:
string is generic constrain, not type of data